src/shared/ui/LongTextInput.tsx

```diff
@@ -0,0 +1,81 @@
+import { StyleSheet, TextInput } from 'react-native';
+
+import { GetProps, setupReactNative, styled } from '@tamagui/core';
+import { focusableInputHOC } from '@tamagui/focusable';
+
+import { IS_IOS } from '@shared/lib';
+
+setupReactNative({
+  TextInput,
+});
+
+export const LongTextInputFrame = styled(
+  TextInput,
+  {
+    name: 'LongTextInput',
+    focusable: true,
+
+    outlineWidth: 0,
+    minWidth: 0,
+    minHeight: 56,
+    maxHeight: 350,
+    maxWidth: '100%',
+    borderWidth: 2,
+    backgroundColor: '$backgroundTransparent',
+    underlineColorAndroid: 'transparent',
+    borderRadius: 4,
+    m: 0,
+    px: 12,
+    py: 8,
+
+    fontSize: 20,
+    selectionColor: 'black',
+
+    multiline: true,
+    textAlignVertical: 'top',
+
+    scrollEnabled: true,
+
+    padding: 8,
+    borderColor: '$mediumGrey',
+
```

Comment:
Please consider css properties ordering:
https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/